### PR TITLE
Sdk passage patch

### PR DESCRIPTION
**Pull request by kevinkowa**
This PR contains the changes for the v8.4.1 release of the Java SDK with a patch to fix PassagesPerDocument for discovery-v2

The tags used for this release are:

git checkout sdk-2020-08-20
Generator: 2.3.1

### CODE REVIEW

**Review by mediumTaj — APPROVED** [review]
👍 looks good!

**Review by mamoonraja — APPROVED** [review]

**Review by germanattanasio — APPROVED** [review]
LGTM

File: discovery/src/test/java/com/ibm/watson/discovery/v1/DiscoveryServiceIT.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ public static void cleanupClass() throws Exception {
   public void setup() throws Exception {
     super.setUp();
     String apiKey = getProperty("discovery.apikey");
-    String url = getProperty("discovery.url");
+    String url = "https://api.us-south.discovery.watson.cloud.ibm.com";
```
Comment (mediumTaj) [inline discovery/src/test/java/com/ibm/watson/discovery/v1/DiscoveryServiceIT.java:149]: Should this be hardcoded? Eventually we should update the credentials to use the proper urls.

**Review by mediumTaj — APPROVED** [review]
👍 looks good! One question but not a big deal....

File: discovery/src/test/java/com/ibm/watson/discovery/v1/DiscoveryServiceIT.java (hunk 1 above)
Comment (kevinkowa) [reply discovery/src/test/java/com/ibm/watson/discovery/v1/DiscoveryServiceIT.java]: updated! [b66e5d4](https://github.com/watson-developer-cloud/java-sdk/pull/1130/commits/b66e5d40c76cf6fdd9e53c4e08060d4b70dfc3cf)

**Review by mediumTaj — APPROVED** [review]
👍 looks good!

**Pull request merged**